src/js/components/AppListItemComponent.jsx

```diff
@@ -20,7 +20,7 @@ var AppListItemComponent = React.createClass({
 
   getLabels: function () {
     var labels = this.props.model.labels;
-    if (labels == null) {
+    if (labels == null || !Object.keys(labels).length) {
```

Comment:
I mean, your solution works too but the `=== 0` makes it more readable imho. We usually do explicit checks and avoid using the `!` inside if checks.